api/v1beta2/integrationtestscenario_webhook.go

```diff
@@ -53,6 +53,28 @@ func (r *IntegrationTestScenario) ValidateCreate() (warnings admission.Warnings,
 					"param manually defined because it will be automatically generated"+
 					"by the integration service")
 		}
+		// we won't enable ITS if git resolver with url & repo+org
+		urlResolverExist := false
+		repoResolverExist := false
+		orgResolverExist := false
+
+		for _, gitResolverParam := range r.Spec.ResolverRef.Params {
+			if gitResolverParam.Name == "URL" && gitResolverParam.Value != "" {
+				urlResolverExist = true
+			}
+			if gitResolverParam.Name == "REPO" && gitResolverParam.Value != "" {
+				repoResolverExist = true
+			}
+			if gitResolverParam.Name == "ORG" && gitResolverParam.Value != "" {
+				repoResolverExist = true
+			}
+			if urlResolverExist && (repoResolverExist || orgResolverExist) {
```

Comment:
This check doesn't check for missing repo or org, if repo+org config is used. It will allow to specify only repo without org.

Reply:
good point @MartinBasti 
I've fixed it to cover the missing use cases
also I ran the unit tests with all scenarios , it works fine for me